src/quo2/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,306 @@
+(ns quo2.components.wallet.transaction-progress.view
```

Comment:
@teoarjun, we should have component tests covering as many variations as possible. Especially since the components have a bunch of conditions. https://github.com/status-im/status-mobile/tree/develop/src/quo2#component-tests